pyaerocom/griddeddata.py

```diff
@@ -1940,7 +1940,10 @@ def crop(self, lon_range=None, lat_range=None, time_range=None, region=None):
         if time_range is None:
             return GriddedData(data, **suppl)
         else:
-            if all(isinstance(x, str) for x in time_range):
+
+            if all(isinstance(x, str) for x in time_range) or all(
+                isinstance(x, np.datetime64) for x in time_range
+            ):
```

Comment:
`isinstance` can take a tuple of classes to check against
``` python
            if all(isinstance(x, (str, np.datetime64)) for x in time_range):
                ...
```
or you really need all of the list elements to be of the same class?